.travis.yml

```diff
@@ -123,6 +124,10 @@ matrix:
         - qt57tools
 before_cache:
   - if [ "${TRAVIS_OS_NAME}" = "osx" ]; then brew cleanup; fi
+# Credit https://discourse.brew.sh/t/best-practice-for-homebrew-on-travis-brew-update-is-5min-to-build-time/5215/9
+# Cache only .git files under "/usr/local/Homebrew" so "brew update" does not take 5min every build
+  - if [ "${TRAVIS_OS_NAME}" = "osx" ]; then find . \! -regex ".+\.git.+" -delete; fi
```

Comment:
Not an expert on find, but don't have to start from `/usr/local/Homebrew` instead of `.`?

Reply:
That makes sense.

Reply:
Feedback applied.